recipes/stompy-ocean/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "stompy-ocean" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stompy_ocean-{{ version }}.tar.gz
+  sha256: 536d8ef35fb8db50bc2dc485c41c6e95e2e94be045113bcdd70726c513fa7416
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
I don't see any extensions that need to be built in this package, and it's marked as `noarch: python`. These are mutually exclusive.